.circleci/config.yml

```diff
@@ -1,42 +1,94 @@
-version: 2.1
 jobs:
-  test:
+  build:
     docker:
-      - image: hysds/pge-base:latest
+      - image: ubuntu:bionic
+        user: root
     steps:
-      - checkout
       - run:
-          name: Test
+          name: Setting Environment Variables
           command: |
-            source $HOME/verdi/bin/activate
-            sudo yum -y update
-            #sudo yum -y install gcc
-            #pip install -U pytest jsonschema
-            mkdir test-reports
-            #pytest --junitxml=test-reports/junit.xml
-
-      - store_test_results:
-          path: test-reports
+            apt-get update
+            apt-get upgrade -y
+            apt-get install -y git
+            apt-get install -y wget
+            # if [ -z ${PYTHONPATH+x} ]; then export PYTHONPATH=""; fi
+            echo $MINTPY_HOME
+            export MINTPY_HOME=~/tools/MintPy
+            export PYTHONPATH=${PYTHONPATH}:${MINTPY_HOME}
+            export PATH=${PATH}:${MINTPY_HOME}/mintpy
+            export PYAPS_HOME=~/tools/PyAPS
+            export PYTHONPATH=${PYTHONPATH}:${PYAPS_HOME}
+      - run:
+          name: Downloading MintPy and PyAPS
+          command: | 
+            export MINTPY_HOME=~/tools/MintPy
+            export PYAPS_HOME=~/tools/PyAPS
+            git clone https://github.com/insarlab/MintPy.git $MINTPY_HOME
+            git clone https://github.com/yunjunz/pyaps3.git $PYAPS_HOME/pyaps3
+            pwd
+            ls
+      - run:
+          name: Installing Dependencies via Conda
+          command: |
+            pwd
+            ls
+            export MINTPY_HOME=~/tools/MintPy
+            export PYTHONPATH=${PYTHONPATH}:${MINTPY_HOME}
+            export PATH=${PATH}:${MINTPY_HOME}/mintpy
+            export PYTHON3DIR=~/tools/miniconda3
+            export PATH=${PATH}:${PYTHON3DIR}/bin
+            export PYAPS_HOME=~/tools/PyAPS
+            export PYTHONPATH=${PYTHONPATH}:${PYAPS_HOME}
```

Comment:
Not a big deal, but could we make this export environment variable shorter and use CONDA_PREFIX instead of PYTHON3DIR, since these are used multiple times here?

```
export MINTPY_HOME=~/tools/MintPy
export PYAPS_HOME=~/tools/PyAPS
export CONDA_PREFIX=~/tools/miniconda3
export PYTHONPATH=${PYTHONPATH}:${MINTPY_HOME}:${PYAPS_HOME}
export PATH=${PATH}:${CONDA_PREFIX}/bin:${MINTPY_HOME}/mintpy
```

Reply:
I just set the environment variables for the above lines in circle CI, as @dbekaert suggested. It would be much cleaner if @mehdizadehm could confirm it works as expected and remove these lines here.